lib/Echidna/Config.hs

```diff
@@ -60,10 +62,15 @@ instance FromJSON EConfig where
                 psender  <- v .:? "psender"  .!= 0x00a329c0648769a73afac7f9381e08fb43dbea70
                 let good = if reverts then (`elem` [ResTrue, ResRevert]) else (== ResTrue)
                 return $ TestConf (good . maybe ResOther classifyRes . view result) (const psender)
-        cc = CampaignConf <$> v .:? "testLimit"   .!= 10000
-                          <*> v .:? "seqLen"      .!= 10
-                          <*> v .:? "shrinkLimit" .!= 5000
-                          <*> pure Nothing
+        cc = do
+          s <- v .:? "seed"
+          let s' :: Maybe StdGen
```

Comment:
do we need the type annotation/auxiliary variable here?

Reply:
fixed